templates/devtest/gitea-ui.tmpl

```diff
@@ -14,14 +14,45 @@
 			<label><input type="checkbox" name="button-state-disabled" value="disabled">disabled</label>
 		</div>
 		<div id="devtest-button-samples">
-			<div>
+			<div class="gt-df gt-fw gt-gap-3">
 				<button class="ui primary button">Primary</button>
+				<button class="ui basic primary button">Basic Primary</button>
 				<button class="ui secondary button">Secondary</button>
 				<button class="ui basic secondary button">Basic Secondary</button>
 				<button class="ui red button">Red</button>
 				<button class="ui basic red button">Basic Red</button>
+				<button class="ui orange button">Orange</button>
+				<button class="ui basic orange button">Basic Orange</button>
+				<button class="ui yellow button">Yellow</button>
+				<button class="ui basic yellow button">Basic Yellow</button>
+				<button class="ui olive button">Olive</button>
+				<button class="ui basic olive button">Basic Olive</button>
+				<button class="ui green button">Green</button>
+				<button class="ui basic green button">Basic Green</button>
+				<button class="ui teal button">Teal</button>
+				<button class="ui basic teal button">Basic Teal</button>
+				<button class="ui blue button">Blue</button>
+				<button class="ui basic blue button">Basic Blue</button>
+				<button class="ui violet button">Violet</button>
+				<button class="ui basic violet button">Basic Violet</button>
+				<button class="ui purple button">Purple</button>
+				<button class="ui basic purple button">Basic Purple</button>
+				<button class="ui pink button">Pink</button>
+				<button class="ui basic pink button">Basic Pink</button>
+				<button class="ui brown button">Brown</button>
+				<button class="ui basic brown button">Basic Brown</button>
+				<button class="ui grey button">Grey</button>
+				<button class="ui basic grey button">Basic Grey</button>
+				<button class="ui black button">Black</button>
+				<button class="ui basic black button">Basic Black</button>
+				<button class="ui negative button">Negative</button>
+				<button class="ui basic negative button">Basic Negative</button>
+				<button class="ui positive button">Positive</button>
+				<button class="ui basic positive button">Basic Positive</button>
 			</div>
+			<div>This is a <button class="ui button tertiary">Tertiary</button> button</div>
 			<div>This is a <button class="ui button button-ghost">Ghost</button> button</div>
+			<div>This is a <button class="ui button button-link">Link</button> button</div>
```

Comment:
What's the differences of these buttons (tertiary/ghost/link)? In which what kind of buttons should be used?

Reply:
Tertiary has neither if the background or underline effects. I wouldn't personally use it anywhere but it's there as an option provided by Fomantic.

Reply:
IIRC it's said that "remove the .ui.tertiary.button and never use it" ?

Reply:
We can't really remove it easily from Fomantic, but I agree, never use it. Remove it from devtest then?

Reply:
For the "tertiary": I think there it could be kept on the page, but add some comments, eg: "deprecated, never use it"

For "ghost": I tested it on my side, the hover background seems quite strange (ugly). I don't think it could be used out-of-box.

For "link": it has underline, but what if developers only want a button with hover but do not want underline? Adding more "gt-xxx" helper seems unnecessary complex.

(Too late and I am going to sleep, will do more tests tomorrow)

Reply:
There is no tertiary button currently in templates, so I'll remove it there. Keeping the styles around just in case.

Reply:
Removed it from devtest